Revit_oM/Misc/RevitRepresentation.cs

```diff
@@ -20,34 +20,35 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-namespace BH.Engine.Adapters.Revit
+using BH.oM.Base;
+using BH.oM.Graphics;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.Revit
 {
-    public static partial class Convert
+    [Description("Fragment containing the representation extracted from Revit element represented by the BHoM object.")]
+    public class RevitRepresentation : IFragment, IImmutable
```

Comment:
@alelom has turned the RenderMesh itself into a fragment here https://github.com/BHoM/BHoM/pull/1047

If you need multiple render meshes, I think this could stay, but if not, you can just add the render mesh as is directly to the BhoMObejct

Reply:
Yeah one object may have more than one `RenderMesh` attached to it, so I will need to stick to the current solution I am afraid.